### PR TITLE
Apposite 20480 c mod11 demo

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. Ready for SME. 

See comments.

### CODE REVIEW

File: Instructions/20480C_MOD11_DEMO.md [hunk 1]
```diff
@@ -2,113 +2,113 @@
...
-5.	In Microsoft Edge, click the **Transformations** button.
-6.	Right-click the web page in Microsoft Edge, and then click **View source**.
-7.	In the source window, locate the &lt;!-- Demonstrate transformations --&lt; comment and review the **&lt;svg&gt;** element: 
-o	The <transform> elements translate the square by 200 units in the X and Y axes, scale it by a factor of 0.5, and rotate it.
+1.	On the **Start** menu, click the **Desktop** tile.
+2.	On the taskbar, click **Microsoft Edge**.
+3.	In Microsoft Edge, open the **Allfiles\Mod11\Democode\SvgDocument.html** file.
+4.	If a message box appears asking if you want to allow blocked content, click **Allow blocked content**.
+5.	In Microsoft Edge, click **Transformations**.
+6.	in Microsoft Edge, right-click the webpage, and then select **View source**.
```
Comment (VinayApposite) [inline Instructions/20480C_MOD11_DEMO.md]: This cannot be done in Edge. This works only in IE. In Edge, you have to use the F12 key to see the source.

File: Instructions/20480C_MOD11_DEMO.md [hunk 2]
```diff
@@ -2,113 +2,113 @@
...
-4.	Hover the mouse over the blue shape on the right side of the window. Verify that the shape changes to a yellow fill color and a dotted green border.
-5.	Move the mouse off the shape. Verify that it reverts to a blue fill color with no outline.
+1.	In Microsoft Edge, click **Events**.
+2.	Place the mouse pointer over the red shape on the left side of the window. Verify that the color of the shape changes to yellow with a dotted green border.
+3.	Move the mouse pointer off the shape. Verify that its color reverts to red with no outline.
+4.	Place the mouse pointer over the blue shape on the right side of the window. Verify that the color of the shape changes to yellow with a dotted green border.
+5.	Move the mouse pointer off the shape. Verify that its color reverts to blue with no outline.
 6.	Click the red shape. Verify that a message box appears, indicating that the shape represents Alaska. Close the message box.
 7.	Click the blue shape. Verify that a message box appears, indicating that the shape represents Hawaii. Close the message box.
-8.	Right-click in the browser window, and then click **View source**.
```
Comment (VinayApposite) [inline Instructions/20480C_MOD11_DEMO.md]: Same comment as earlier.

File: Instructions/20480C_MOD11_DEMO.md [hunk 3]
```diff
@@ -2,113 +2,113 @@
...
-4.	If a message box appears asking if you want to allow blocked content, click the **Allow blocked content** button.
-5.	In Microsoft Edge, click the **Separate Transformations** button.
-6.	Right-click the web page in Microsoft Edge, and then click **View source**.
-7.	In the source window, locate the **demoSeperateTransformations** function and review the code: 
+1.	On the **Start** menu, click the **Desktop** tile.
+2.	On the taskbar, click **Microsoft Edge**.
+3.	In Microsoft Edge, open the **Allfiles\Mod11\Democode\CanvasDocument.html** file.
+4.	If a message box appears asking if you want to allow blocked content, click **Allow blocked content**.
+5.	In Microsoft Edge, click **Separate Transformations**.
+6.	In Microsoft Edge, right-click the webpage, and then select **View source**.
```
Comment (VinayApposite) [inline Instructions/20480C_MOD11_DEMO.md]: Same comment as earlier.

**Comment by barucht** [comment]
Ready for QA

**Pull request merged**